src/features/render/lib/phaser/scenes/WorldScene.tsx

```diff
@@ -406,12 +406,20 @@ export class WorldScene extends WarpableScene {
     profiles: Array<ProfileInfo>,
   ) {
     Object.keys(entityUpdates).forEach((entityId) => {
-      // Ignore player character
-      if (entityId === this.playerAddress) return;
 
       const entityUpdate = entityUpdates[entityId];
       if (entityUpdate.Type === "Avatar") {
         const spriteKeyBase = this.spriteKeyBase(entityId, entityUpdate);
+        if (entityId === this.playerAddress) {
+          // Update the player's sprite key if the skin has changed
+          if (spriteKeyBase !== this.playerSpriteKeyBase) {
+            this.playerSpriteKeyBase = spriteKeyBase;
+            // Update the player's animation with the new skin
+            const playerSprite = this.player.getAt(0) as Phaser.GameObjects.Sprite;
+            playerSprite.play(`${this.playerSpriteKeyBase}_idle`);
```

Comment:
This could be removed, but at the moment it requires the player to move for this to take effect, this forces that faster.

If the player specific logic continuously sent requests to play walk or play idle over and over even if nothing had changed this would not be needed.